src/main/java/io/vertx/amqp/AmqpSender.java

```diff
@@ -83,4 +83,9 @@
    * @return the connection having created the sender.
    */
   AmqpConnection connection();
+
+  /**
+   * @return the remaining credit, 0 is none.
+   */
+  long getRemainingCredit();
```

Comment:
should be named `remainingCredits()`

Reply:
Without the `get`?

Reply:
yes that's the convention in Vert.x when it's a read-only property, e.g `connection()` in the same interface

Reply:
Sorry, forgot about these (every project has different conventions these days, we should write a manifesto about that :-))